pytest_fixtures/content_hosts.py

```diff
@@ -8,99 +8,78 @@
 from broker.broker import VMBroker
 
 from robottelo.config import settings
-from robottelo.constants import BROKER_RHEL77
 from robottelo.hosts import ContentHost
 
 
 @pytest.fixture
-def rhel_contenthost(request):
+def host_conf(request):
+    """A function-level fixture that returns parameters for VMBroker host deployment"""
+    conf = params = {}
+    if hasattr(request.node, 'callspec'):
+        params = [*request.node.callspec.params.values()][0]
+    conf['workflow'] = params.get('workflow', settings.content_host.deploy_workflow)
+    _rhelver = f"rhel{params.get('rhel_version', settings.content_host.default_rhel_version)}"
+    conf['rhel_ver'] = settings.content_host.hardware.get(f'{_rhelver}').release
```

Comment:
can't you directly access the `settings.content_host.hardware.get(_rhelver).release` as _rhelver is string object seems